docs/introduction.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Introduction
+sidebar_position: 0
+---
+
+
+# Introduction
+
+:::warning
+Check out our [Quick Start](./quickstart) guide to get started in a snap.
+:::
+
+Blindnet offers a simple solution to enable data privacy and compliance with data minimization requirements in various data regulations, including GDPR and CCPA / CPRA. Beyond compliance, we provide software developers with a zero-trust service to minimize the exposure to unnecessary user data and reduce the risk in case of data breaches, via two complementary solutions:
+
+1. **blindnet devkit**, offering end-to-end encryption in your web applications ;
```

Comment:
See https://github.com/blindnet-io/devrel-management/issues/7#issuecomment-1153675417

But it is indeed too soon to put this on blindnet.dev, I just tried to improve what was already written without changing it too much. Could you please suggest an alternative text ?

Reply:
I removed other mentions of the "Privacy Stack". Yet, I'm leaving this one to act as a "teaser" for our future SaaS offering. @filipblnt let me know if this seems OK to you.

Reply:
Well, I still think there is only one solution. Maybe different wordings for different audience, but as of today we only have one product planned and it is the devkit. I would use the same text as in the comment I linked, at least for now, because devkit is much more than just e2ee.